01-introduction.Rmd

```diff
@@ -264,13 +283,10 @@ knitr::kable(top_dls[1:5, 1:2], digits = 0,
 # cranlogs::cran_top_downloads(when = "last-month") # most downloaded pkgs
 ```
 
-A parallel group of developments relates to the [rspatial](https://github.com/rspatial) set of packages.^[Note the difference between "r-spatial", organization containing packages such as **sf**, and "rspatial", organization responsible for **terra**.]
```

Comment:
@Robinlovelace maybe it would be good to keep the footnote about the rspatial/r-spatial difference

Reply:
Agree it's potentially useful info. Can be found out online and not sure it lives in the book. No big objections to re-adding it or strong feelings, just felt a bit like too much info to me and footnotes are fiddly and generally discouraged I think.

Thoughts @jannes-m to get another opinion?